packages/odata-v4/src/uri-conversion/uri-value-converter.ts

```diff
@@ -1,7 +1,4 @@
 import { createUriConverter } from '@sap-cloud-sdk/odata-common/internal';
 import { defaultDeSerializers } from '../de-serializers';
 
-/**
- * internal
- */
 export const uriConverter = createUriConverter(defaultDeSerializers);
```

Comment:
The previous comment was obvisouly broken, but shouldn't it still complain about missing `@internal` if it is internal or any documentation if there is no documentation?
Or is this something that still needs to be done?

Reply:
Yes, this still needs to be done. Actually the require-jsdoc rule be default only checks functions and classes and doesn't warn for missing docs on interface, enum, types, etc. I'll add more `contexts` to expand it.